features/nvstore/source/nvstore.h

```diff
@@ -422,14 +414,14 @@ class NVStore : private mbed::NonCopyable<NVStore> {
     /**
      * @brief Actual logics of set API (covers also set_once and remove APIs).
      *
-     * @param[out] key                    key (both input and output).
+     * @param[in]  key                    key.
      * @param[in]  buf_size               Buffer size (bytes).
      * @param[in]  buf                    Input Buffer.
      * @param[in]  flags                  Record flags.
      *
      * @returns 0 for success, nonzero for failure.
      */
-    int do_set(uint16_t &key, uint16_t buf_size, const void *buf, uint16_t flags);
+    int do_set(uint16_t key, uint16_t buf_size, const void *buf, uint16_t flags);
```

Comment:
This is also a breaking change.

Reply:
That's not a breaking change, as the function is private (maybe the doxy comments are confusing to think it is not).

Reply:
Ah, I missed the private at the top. Thanks.